scripts/qa_crowbarsetup.sh

```diff
@@ -1899,7 +1899,9 @@ EOH
         yaml_osds=`echo $cephosds | sed "s/ /\n/g" | sed "s/\..*//g" | sort -ru`
         # for radosgw, we only want one node, so enforce that
         yaml_radosgw=`echo $cephradosgws | sed "s/ .*//g" | sed "s/\..*//g"`
-        ceph_version=`rpm -q --qf %{version} ceph`
+
+        ceph_nodes=($yaml_allnodes)
+        ceph_version=`ssh ${ceph_nodes[0]} \"rpm -q --qf %{version} ceph\"`
```

Comment:
the \" seem wrong. It makes the params part of the command name.
with just " or without quotes, it works.